scripts/ci/publish.sh

```diff
@@ -10,7 +10,7 @@ fi
 
 function image_push() {
     echo ${DOCKERHUB_TOKEN} | docker login --username sonobuoybot --password-stdin
-    make containers push
```

Comment:
In CI the images will be created in another step (linux+windows) and the publishing step will just need to push.